clients/cobol-lsp-vscode-extension/src/services/settings/util/SettingsUtils.ts

```diff
@@ -0,0 +1,30 @@
+import * as vscode from "vscode";
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+export class SettingsUtils {
+    public static isValidJSON(json: string): boolean {
+        return (json !== undefined && json.trim().length > 0);
```

Comment:
It's better to check it by parsing https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse

Reply:
makes sense 👍